### DISCUSSION TITLE
way to toggle Funnel combo points

### DISCUSSION THREAD

**Discussion by Bigsack** [Hekili/hekili#1340] [Q&A]
Hi i was wondering how would i be able to toggle the funnel option on sub rogue rotation without having to click it in the mini map icon ?

**Comment by Hekili** [comment]
You can update (most) specialization options using `/hekili set`.  You could make a macro for this if you wanted.

`/hekili set priority_rotation` would toggle it.

`/hekili set priority_rotation on` would turn it on.
`/hekili set priority_rotation off` would turn it off.

You have to use the exact name of the setting for it to work (a current bug that I'll resolve).

`/hekili set` will list the options.
